hack/generate/csv.sh

```diff
@@ -37,12 +38,19 @@ function kafka_image {
 
 image "queue-proxy"    "${serving}-queue"
 image "activator"      "${serving}-activator"
+image "activator__kube-rbac-proxy"  "$rbac_proxy"
```

Comment:
Potential refactor once this is in: We could write a special transformer that picks up an `IMAGE_kube-rbac-proxy` variable and just replaces that in every container that's named `kube-rbac-proxy` to not have to repeat this a gazilion times.

Reply:
Yeah I thought about so I was not sure about where to have that. I saw that the manifest is appended at the operator side at reconcile phase. If a transformer at the extension side works then yes.

Reply:
Yeah, transformer on the extension side should work.